proto_defs/src/control/config.pb.h

```diff
@@ -47,6 +47,13 @@ typedef enum _particle_ctrl_Feature {
     particle_ctrl_Feature_ETHERNET_DETECTION = 1 /* Ethernet FeatherWing detection on boot */
 } particle_ctrl_Feature;
 
+/* Note: Use the same values as defined by the security_key_type enum */
+typedef enum _particle_ctrl_SetProtectedStateRequest_Action { 
+    particle_ctrl_SetProtectedStateRequest_Action_RESET = 0, /* Invalid key */
+    particle_ctrl_SetProtectedStateRequest_Action_DISABLE_REQUEST = 1, /* TCP device private key */
+    particle_ctrl_SetProtectedStateRequest_Action_DISABLE_CONFIRM = 2 /* TCP device public key */
```

Comment:
These comments are weird. Where do they come from?

Reply:
Yeah, saw it in the diff too. It's most likely a bug in the nanopb generator :shrug: